src/views/add-boiler-house-view/add-boiler-house-view.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { useParams } from "react-router-dom";
+
+import { AddBoilerHouseLayout } from "./layout";
+
+import { useAsset } from "@mtfh/common/lib/api/asset/v1";
+import { Center, Spinner } from "@mtfh/common/lib/components";
+
+export const AddBoilerHouseView = (): JSX.Element => {
+  const { assetId } = useParams<{ assetId: string }>();
+
+  const { data: asset } = useAsset(assetId);
+
+  if (asset === undefined) {
```

Comment:
What if the asset is not retrieved at all? Unlikely but possible

Reply:
Youre right, I will add better error handling